.rubocop.yml

```diff
@@ -125,7 +125,6 @@ Style/FormatStringToken:
   Exclude:
     - "rakelib/mvi.rake"
 
-
 # Disabling this rule for now. We should consider changing.
```

Comment:
Can we remove this comment?  We're not disabling it.  We're just excluding one file (which has a comment/explanation in line)

Reply:
Yeah, there is also a comment on the cop just above it as well where they mention disabling it. I'll remove from both.